src/google/protobuf/descriptor.cc

```diff
@@ -5480,6 +5489,75 @@ void DescriptorBuilder::BuildMessage(const DescriptorProto& proto,
   }
 }
 
+std::string GetJSONName(const FieldDescriptorProto& field, bool use_custom, bool* was_custom) {
```

Comment:
JSON -> Json. Acronyms should only capitalize the first letter.

Reply:
Roger that. (I mostly write Go, which uses different conventions for acronyms than typical C++ and Java norms. Mea culpa.)

Reply:
Done